Account/FxADevice.swift

```diff
@@ -5,32 +5,43 @@
 import Foundation
 import Shared
 
+typealias PushParams = (callback: String, publicKey: String?, authKey: String?)
```

Comment:
Nit: Looks like you've defined this to clean up the parameters for the initializer but then break out each property into an instance property on `FxADevice` below. Maybe make this a struct and declare it as instance variable?